packages/office-ui-fabric-react/src/DragDrop.ts

```diff
@@ -0,0 +1 @@
+export * from 'office-ui-fabric-react/lib/DragDrop';
```

Comment:
Can you make similar files in `react` and `react-next`?

Reply:
done.